src/index.js

```diff
@@ -1,14 +1,81 @@
 import './style.css';
-import _ from 'lodash';
 
- function component() {
-   const element = document.createElement('div');
+const todoTasks = [
+  {
+    description: 'To do 1',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'to do 2',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'to do 3',
+    completed: true,
+    index: 2,
+  },
+];
 
-  // Lodash, now imported by this script
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-   element.classList.add('hello');
-   
-   return element;
- }
+document.addEventListener('DOMContentLoaded', () => {
+  const todoList = document.getElementById('todo_section');
 
- document.body.appendChild(component());
\ No newline at end of file
+  const form = document.createElement('form');
+  form.classList.add('form');
+
+  const header = document.createElement('div');
+  header.classList.add('header');
+  const heading = document.createElement('span');
+  heading.textContent = 'Today\'s To Do';
+  header.appendChild(heading);
+  const reloadIcon = document.createElement('span');
+  reloadIcon.classList.add('material-symbols-outlined');
+  reloadIcon.textContent = 'cached';
+  header.appendChild(reloadIcon);
+  const input = document.createElement('input');
+  input.classList.add('input');
+  input.placeholder = 'Add to your list...';
+  form.appendChild(header);
+  form.appendChild(input);
+  todoList.appendChild(form);
+
+  const displayTodoList = () => {
+    const unorderedList = document.createElement('ul');
+    unorderedList.classList.add('unorderedList');
+
+    todoTasks.forEach((task) => {
+      const listItem = document.createElement('li');
+      listItem.classList.add('listItem');
+
+      const descriptionSection = document.createElement('div');
+      descriptionSection.classList.add('descriptionSection');
+
+      const checkBox = document.createElement('span');
+      checkBox.classList.add('material-symbols-outlined');
+      checkBox.textContent = 'check_box_outline_blank';
+
+      const description = document.createElement('p');
+      description.textContent = task.description;
```

Comment:
- [ ] [OPTIONAL] You could try to make the code cleaner by using the power of template literals while iterating through the tasks

something like this,
```

  toDoItemsArray.forEach((elem, index) => {
    markup += `<section class="to-do__item  border-bottom" id=${index}>
    <form class="to-do__edit hide flex-row border-bottom">
    <input type="text" class="to-do__input" placeholder="Input a task or use delete button..."/>
    <button>
    <span class="material-symbols-outlined to-do__addTask">add_task</span>
    </button>
    </form>
    <div class="flex-row to-do__task">
    <input class='checkbox' type="checkbox" ${elem.completed && 'checked'}/>
    <p class='description ${elem.completed && 'completed'}'>${elem.description}</p>
    <span class="to-do__item-icons">
    <span class="material-symbols-outlined dltTaskBtn" id=${index}>delete</span>
    <span class="material-symbols-outlined editTaskBtn" id=${index}>edit_note</span>
    </span>
    </div>
  </section>`;
  });

  toDoItemsEl.innerHTML = markup;
};
```

As you can see, it is more relatable, because the HTML code is used directly 😉